web/templates/compose_banner/automatic_new_visibility_policy_banner.hbs

```diff
@@ -0,0 +1,10 @@
+{{#> compose_banner }}
+    <p class="banner_message">
+        {{#if followed}}
+            {{#tr}}Now following {{/tr}}
+        {{else}}
+            {{#tr}}Unmuted {{/tr}}
+        {{/if}}
+        <a class="above_compose_banner_action_link" href="{{narrow_url}}" data-message-id="{{link_msg_id}}">{{stream_topic}}</a>.
```

Comment:
Translation tags don't like work like this -- because other languages have a different word order from English, you always need to have the entire sentence tagged for translation. Grep for `z-link` to see how this is done properly.

This is probably a good time to carefully read https://zulip.readthedocs.io/en/latest/translating/internationalization.html

Reply:
#27286